docs/en-US/about_Plaster_CreatingAManifest.help.md

```diff
@@ -97,7 +111,7 @@ Additionally, a default value can be specified, as shown in the next example:
 
 This results in the following output, with the default value in parentheses:
 ```
-Enter the version number for the module (1.0.0):
+Enter the version number for the module (0.1.0):
```

Comment:
Should this be 0.1.0  or 1.0.0?  My original thinking was start them at 0.1.0 as they develop the template.  But it is highly likely whatever we put in as a version is what it will ship with.